packages/esm-patient-programs-app/src/programs/programs-form.component.tsx

```diff
@@ -120,7 +121,7 @@ const ProgramsForm: React.FC<ProgramsFormProps> = ({
             (response) => {
               if (response.status === 200) {
                 mutateEnrollments();
-                closeWorkspace();
+                closeWorkspaceWithSavedChanges();
```

Comment:
Hi @jayasanka-sack!
This solves the failing e2e test for programs.

The reason for the failing test was that the on saving the form, the UI was blocked by the prompt, which is resolved in this PR.
Thanks!
![image](https://github.com/openmrs/openmrs-esm-patient-chart/assets/51502471/283ca7bd-9528-46cc-ae30-1879f9c4a140)